pkg/virt-controller/watch/job.go

```diff
@@ -0,0 +1,69 @@
+package watch
+
+import (
+	kubeapi "k8s.io/client-go/pkg/api"
+	batchv1 "k8s.io/client-go/pkg/apis/batch/v1"
+	"k8s.io/client-go/pkg/fields"
+	"k8s.io/client-go/pkg/labels"
+	"k8s.io/client-go/pkg/util/workqueue"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/tools/record"
+	kvirtv1 "kubevirt.io/kubevirt/pkg/api/v1"
+	"kubevirt.io/kubevirt/pkg/kubecli"
+	"kubevirt.io/kubevirt/pkg/virt-controller/services"
+)
+
+func migrationJobSelector() kubeapi.ListOptions {
+	fieldSelector := fields.Everything()
+	labelSelector, err := labels.Parse(kvirtv1.DomainLabel)
+	if err != nil {
+		panic(err)
```

Comment:
We probably don't want to panic here. make this function return

(kubeapi.ListOptions, error)

and handle the error higher in the call chain.

Also, we definitely want to log this error.

logger := logging.DefaultLogger()